docs/using-gitbase/getting-started.md

```diff
@@ -0,0 +1,72 @@
+# Getting started
+
+## Prerequisites
+
+**gitbase** has two optional dependencies that should be running on your system if you're planning on using certain functionality.
```

Comment:
I think this can be left for a later review but I would write "gitbase" always the same way. Either bold, italic or preformatted. It now appears in different ways in the documentation.